prod/westeurope/internal/api/functions_admin/function_app/terragrunt.hcl

```diff
@@ -72,6 +85,15 @@ inputs = {
     AZURE_APIM                = "io-p-apim-api"
     AZURE_APIM_HOST           = "api-internal.io.italia.it"
     AZURE_APIM_RESOURCE_GROUP = "io-p-rg-internal"
+
+    MessageContentStorageConnection = dependency.storage_account.outputs.primary_connection_string
```

Comment:
Duplicate of StorageConnection above. No need to create a new var.

Reply:
We refer to this storage as `MessageContentStorageConnection` in the application. This is to differentiate storages that may eventually be different. @gunzip do we need to change the service implementation?

Reply:
yes that would be better to rename it to `StorageConnection` if possible